bigtop-tests/test-artifacts/giraph/src/main/groovy/org/apache/bigtop/itest/giraphsmoke/TestGiraphSmoke.groovy

```diff
@@ -24,17 +24,12 @@ import org.apache.bigtop.itest.shell.Shell
 import org.junit.Test
 
 public class TestGiraphSmoke {
-  static String runnerScript = "HADOOP_CLASSPATH=/etc/giraph/conf hadoop jar"
   static String testDir = "test.giraphsmoke." + (new Date().getTime())
-
-  static String giraphHome = System.getProperty('GIRAPH_HOME', '/usr/lib/giraph')
-  static String giraphJar = "${giraphHome}/giraph-jar-with-dependencies.jar"
-
   static Shell sh = new Shell("/bin/bash -s")
 
   @Test(timeout = 300000L)
   public void testPageRankBenchmark() {
-    sh.exec("${runnerScript} ${giraphJar}"
+    sh.exec("hadoop jar /usr/lib/giraph/giraph-examples-1.1.0-for-hadoop-2.8.4-jar-with-dependencies.jar"
```

Comment:
Shall we use * in this test. Otherwise it can only survive for these specific versions.

Reply:
ok, I'll modify it.